libcxx/test/libcxx/mangled_names.pass.cpp

```diff
@@ -0,0 +1,74 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// We're using `string::starts_with` in this test
+// UNSUPPORTED: c++03, c++11, c++14, c++17
```

Comment:
It seems more important that we test the manglings are consistent across dialects,  rather than use `starts_with`.